src/Components/Aspire.Azure.Security.KeyVault/README.md

```diff
@@ -63,10 +63,10 @@ The .NET Aspire Azure Key Vault library provides multiple options to configure t
 
 ### Use a connection string
 
-When using a connection string from the `ConnectionStrings` configuration section, you can provide the name of the connection string when calling `builder.AddAzureKeyVaultSecrets()`:
+When using a connection string from the `ConnectionStrings` configuration section, you can provide the name of the connection string when calling `builder.AddAzureKeyVault()`:
 
 ```csharp
-builder.AddAzureKeyVaultSecrets("secretConnectionName");
+builder.AddAzureKeyVault("secretConnectionName");
```

Comment:
```suggestion
builder.AddAzureKeyVaultClient("secretConnectionName");
```

Reply:
I don't agree, this location is about the AppHost, all other READMEs do the same here.

Reply:
>  this location is about the AppHost,

That is not correct.

Reply:
I'll fix it in all the READMEs then